src/js/zqd/zqd.js

```diff
@@ -124,6 +124,12 @@ export class ZQD {
       "-zeekrunner",
       zeekRunnerCommand(this.zeekRunner)
     ]
+
+    if (process.platform !== "win32") {
+      const {readfd} = require("node-pipe").pipeSync()
+      opts.stdio.push(readfd)
+      args.push("-brimfd", opts.stdio.length - 1)
```

Comment:
```suggestion
      args.push("-brimfd", opts.stdio[opts.stdio.length - 1])
```

Reply:
I originally thought this to be the case as well but because you pass the readfd as the fourth arg in `stdio`, the fd on the newly created process ends up being 4.